infra/label_validation_agent_step_functions/handlers/load_batches.py

```
@@ -0,0 +1,84 @@
+        # Each ValidateLabels Lambda will read its specific batch from the manifest using the index
+        result = {
+            "batch_indices": list(range(total_batches)),
+            "manifest_s3_key": manifest_key,
+            "total_batches": total_batches,
+        }
+        # Pass through langchain_project if provided
+        if langchain_project:
+            result["langchain_project"] = langchain_project
+        return result
+    except Exception as e:
+        logger.error("Failed to load batches from %s: %s", manifest_key, e)
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)